site/src/pages/ManagementSettingsPage/OrganizationSettingsPage.test.tsx

```diff
@@ -0,0 +1,125 @@
+import { screen, within } from "@testing-library/react";
+import { HttpResponse, http } from "msw";
+import {
+  MockDefaultOrganization,
+  MockOrganization2,
+} from "testHelpers/entities";
+import {
+  renderWithManagementSettingsLayout,
+  waitForLoaderToBeRemoved,
+} from "testHelpers/renderHelpers";
+import { server } from "testHelpers/server";
+import OrganizationSettingsPage from "./OrganizationSettingsPage";
+
+jest.spyOn(console, "error").mockImplementation(() => {});
+
+const renderRootPage = async () => {
+  renderWithManagementSettingsLayout(<OrganizationSettingsPage />, {
+    route: "/organizations",
+    path: "/organizations",
```

Comment:
pretty sure you can just change this to...
```suggestion
    path: "/organizations/:organization?",
```

...and then get rid of the `extraRoutes`

Reply:
Oh clever!